src/util/tracking.ts

```diff
@@ -80,15 +82,16 @@ async function logToFirebase(name: TrackingEventName, values: TrackingValues) {
   if (installerId != null) params.aid = installerId
   if (pluginId != null) params.plugin = pluginId
   if (error != null) params.error = error
-  // @ts-expect-error
-  global.firebase.analytics().logEvent(name, params)
```

Comment:
You are changing logic which is not a part of the task. For now keep both log events.